docker/audit-docker-compose.mongo.yml

```diff
@@ -0,0 +1,31 @@
+version: '2'
+services:
+  activiti-cloud-audit:
```

Comment:
All this entry would be moved to `infrastructure-docker.yml` ans commented code. See my previous comment.